src/main/java/net/openhft/chronicle/wire/BinaryWire.java

```diff
@@ -3560,7 +3559,7 @@ public boolean isNull() {
 
         @Override
         @Nullable
-        public Object marshallable(@Nullable Object object, @NotNull SerializationStrategy strategy)
+        public Object marshallable(@NotNull Object object, @NotNull SerializationStrategy strategy)
```

Comment:
This is consistent with the other implementations of this, and the places that it's called. I added these annotations to the interface as well for consistency.